.github/ISSUE_TEMPLATE.md

```diff
@@ -1,4 +1,16 @@
 <!-- Thank you for reporting! -->
-
 <!-- Please go to the forum first for support with crashes and issues. -->
 <!-- http://www.minecraftforge.net/forum/forum/18-support-bug-reports/ -->
+
+**Minecraft Version:** {mc version}
+
+**Forge Version:** {forge version}
+
+**Full Log:** {link to gist/pastebin with full latest.log}
```

Comment:
For GitHub issues, Gist is probably preferred, as it allows larger file sizes and I think stays valid for longer (the poster can be assumed to have the required account, for obvious reasons).